spring-boot-project/spring-boot/src/main/java/org/springframework/boot/logging/logback/DefaultLogbackConfiguration.java

```diff
@@ -75,11 +76,13 @@ private void defaults(LogbackConfigurator config) {
 		String defaultCharset = Charset.defaultCharset().name();
 		config.getContext().putProperty("CONSOLE_LOG_CHARSET",
 				resolve(config, "${CONSOLE_LOG_CHARSET:-" + defaultCharset + "}"));
+		config.getContext().putProperty("CONSOLE_LOG_THRESHOLD", resolve(config, "${CONSOLE_LOG_THRESHOLD:-TRACE}"));
```

Comment:
Isn't the default `DEBUG` rather than `TRACE`?

Reply:
It's filter level of appender, not log level of logger,  `TRACE` means appender won't drop any log from logger by default. For example,  `log.debug(...)` emit a debug level log, if the logger's log level is DEBUG, this log won't be dropped,  then the log is passed to file appender and console appender,  if filter level of file appender is TRACE, this log is written to file, if filter level of console appender is INFO, this log won't be written to console.